Automattic/wp-calypso
# Plugins Multisite - Update the plugin autoupdate tooltip visibility

#### Proposed Changes

Update the plugin autoupdate tooltip visibility:
- Change the hiding mode of the label to `visibility: hidden` instead of `display: none`
- Update the tooltip to be always visible, even when the label is invisible
- Update the `z-index` to allow the tooltip to be shown on the modal

#### Testing Instructions



* Go to a multisite plugins page. Ex: `/plugins/woocommerce-subscriptions` PS: no site at the end of the URL
* Make sure you install or have it installed in at least one site
* Click on manage sites
* Check if the autoupdate option
  * is disabled
  * have a `i` icon on the side
  * you can see a explanation why it is disabled after clicking on the `i`

| Before  | After |
| ------------- | ------------- |
|<img width="949" alt="Screen Shot 2022-09-19 at 10 20 04" src="https://user-images.githubusercontent.com/5039531/191053126-8bb7d53b-63a5-4ecd-a367-fabea82014bb.png">|<img width="949" alt="Screen Shot 2022-09-19 at 10 20 24" src="https://user-images.githubusercontent.com/5039531/191053190-20e5931c-ae58-4680-8d48-9cc784e8853e.png">|



#### Pre-merge Checklist






---

Fixes #67541
